src/documentation/51.javascript-sdk.md

```diff
@@ -90,36 +90,26 @@ console.log(result); // [ 123456 ] (The ID of the shipment that was just created
 
 Our SDK exposes some constants to make working with our API easier.
 
-- Carriers
+- **Carriers**
   ```js
-  import { CARRIERS } from '@myparcelnl/sdk';
+  import { CarrierName } from '@myparcelnl/constants';
   
-  CARRIERS.POSTNL_NAME // "postnl"
+  CarrierName.PostNL // "postnl"
   ```
-- **Package types:** Contains all package types' names and IDs.
+- **Package types**
   ```js
-  import { PACKAGE_TYPES } from '@myparcelnl/sdk';
+  import { PackageTypeId, PackageTypeName } from '@myparcelnl/constants';
   
-  PACKAGE_TYPES.DIGITAL_STAMP_NAME // "digital_stamp"
-  PACKAGE_TYPES.PACKAGE_ID // 1
-  PACKAGE_TYPES.LETTER // { ID: 3, NAME: "letter" }
+  PackageTypeName.DigitalStamp // "digital_stamp"
+  PackageTypeId.Package // 1
   // etc
   ```
 - **Delivery types**
   ```js
-  import { DELIVERY_TYPES } from '@myparcelnl/sdk';
+  import { DeliveryTypeId, DeliveryTypeName } from '@myparcelnl/constants';
 
-  DELIVERY_TYPES.STANDARD_NAME // "standard"
-  DELIVERY_TYPES.PICKUP_ID // 4
-  DELIVERY_TYPES.MORNING // { ID: 1, NAME: "morning" }
-  // etc
-  ```
-- **Countries:** Contains constants for all countries, by name.
-  ```js
-  import { COUNTRIES } from '@myparcelnl/sdk';
-  
-  COUNTRIES.NETHERLANDS // "NL"
-  COUNTRIES.GERMANY // "DE"
+  DeliveryTypeName.Standard // "standard"
+  DeliveryTypeId.Pickup // 4
   // etc
   ```
```

Comment:
Countries are exposed through `@myparcelnl/constants/countries`, we've decided to split up countries, states and utils this way:

```ts
import {NETHERLANDS, BELGIUM, ALL_COUNTRIES, EU_COUNTRIES} from '@myparcel/constants/countries';
import {GEORGIA} from '@myparcel/constants/states';
import {toPackageTypeId} from '@myparcel/constants/helpers';

const mailboxId = toPackageTypeId(PackageTypeName.Mailbox); // 2
```

This reduces clutter when searching for a const through a suggestions menu in an IDE, and removes ambiguity between for example the country Georgia, and the state Georgia in the USA.

The only caveat for this (sadly) is you [*need* `moduleResolution` to be `Node16` or `NodeNext` in `tsconfig.json`](https://github.com/microsoft/TypeScript/issues/51862#issuecomment-1358049778).

It would be great if you could restore + update that piece of documentation as well. 😄 You don't have to add anything on the `tsconfig.json` thing, we can do that ourselves after this is merged.

Reply:
Sounds good, will push a fix later today!

Reply:
Just pushed a fix that adds some countries and states examples